include/util.js

```diff
@@ -482,7 +482,7 @@ Util.addEvent = function (obj, evType, fn) {
         obj.addEventListener(evType, fn, false);
         return true;
     } else {
-        throw new Error("Handler could not be attached");
+        throw new Error("Handler for \"" + evType + "\" on object \"" + obj + "\" could not be attached");
```

Comment:
Nit: instead of escaping the double quotes, it's probably more readable to just use single quotes as the outer quotes.